extensions/aws2-ddb/runtime/src/doc/configuration.adoc

```diff
@@ -0,0 +1,60 @@
+There are 2 ways of DynamoDB client creation:
```

Comment:
What would think about the wording below ? If I get it right at all :)
The aws2-ddb extension is using a DynamoDB client under the hood. The underlying DynamoDB client could either be created by this extension itself or by the quarkus-amazon-dynamobdb.

Reply:
Nice:) this explains the reality much better, I'll use it

Reply:
done